### DISCUSSION TITLE
suggestion to contribute on hacktoberfest 2020, quality of PR

### DISCUSSION THREAD

**Discussion by NirmalSilwal** [EddieHubCommunity/support#291] [General] [answered]
Hi contributor there,

I find many people making PR on repo like [this](https://github.com/bhaveshlohana/HacktoberFest2020-Contributions) and [this one](https://github.com/Cullyege/Hacktoberfest2020). Is it a good idea to make PR on such project or violation of quality of PR in hactoberfest?

Looking for suggestion.

Thanks

**Comment by eddiejaoude** [comment, marked as answer]
These type of practice PRs, are good for people getting started with the workflow, but I think will be excluded from hacktoberfest. 

Hacktoberfest is to encourage people to get into open source and add value to a project and community. PRs do not need to be big, but they need to add value.

I suggest find repos that you want to contribute to and add value, for example, filling in the gaps in the documentation
